mkdocs.yml

```diff
@@ -110,8 +111,15 @@ extra:
       link: https://dietpi.com/
     - icon: fontawesome/brands/twitter
       link: https://twitter.com/DietPi_
+  item076:
+      name: LAMP Webstack
+      display_name: LAMP Webstack - Apache2 / MariaDB / PHP
+      long_desc: LAMP stack is a popular open source web platform commonly used to run dynamic web sites and servers. It is considered by many, as the platform of choice for development and deployment of high performance web applications which require a solid and reliable foundation.
+      docs_url: software/webserver_stack/#lamp-webstack
+      web_note: DietPi will automatically install a web server stack if any software requires it. By default this is Lighttpd, however, you can change this in the DietPi-Software menu.
+      web_image: images/dietpi-software/lighttpd_0.jpg 
```

Comment:
Mid-term, we should split the images to cover `dietpi-software.html`, like:
```
logo:
screenshot:
```
Just a single "fullscreen" screenshot should be okay. Actually I wanted to merge those in `dietpi-software.html`, so that when opening a software details slide, the full screenshot is loaded directly. It takes slightly longer to load but allows making the slides larger for large screens and when the image is clicked, no second file needs to be loaded. Also it reduces overhead for us to create not only a screenshot but additionally a smaller version of it.

Reply:
included